README.md

```diff
@@ -190,6 +190,22 @@ None.
         password: similarly-secure-password
         priv: "example_db.*:ALL"
 
+## Replication Master Master
+
+Variable to define to use Master Master replication on each host:
+
+    mysql_logs_bin:
+    mysql_replication_master:
+    mysql_replication_role: [master01] / [master02]
+    mysql_server_id:
+
+to use it:
+
+    - hosts: Databases-Servers
+      become: yes
+      roles:
+        - { role: ansible-role-mysql, mysql_replication_master_master: true}
```

Comment:
Booleans are not necessarily the best idea, I would prefer `mysql_replication_mode: comaster|masterslave` for example. It is easier if other methods are added later and reduce the overall number of variables.